docs/odcs.rst

```diff
@@ -46,46 +46,49 @@ information for requesting a compose to be created by ODCS::
     compose:
       # Optional, default False. Inherit compose used for building parent image
       inherit: true
-      # Optional, default "tag".
-      type: tag
-      # Required for "tag" type. Must contain at least one item.
+      # Required for ODCS Koji "tag" type usage. Must contain at least one item.
       packages:
       - package-1-name # This is a package name, not an NVR.
       - package-2-name
-      # Optional. Default value is configured in OSBS environment
-      sigkeys:
-      - AB123456
+      # Optional. Default and possible values are configured in OSBS environment
+      # which will be a mapping of intent -> list of signing keys
+      #     release -> my-release-key
+      #     beta -> my-beta-key, my-release-key
+      #     unsigned -> <empty>
+      intent: release
 
 Alternatively, for modules support::
 
     compose:
-      type: module
       # Eventually, ODCS will support ":" separator, for now "-" is used:
       #     https://pagure.io/odcs/issue/98
-      # Required for "module" type. Must contain at least one item.
+      # Required for ODCS "module" type usage. Must contain at least one item.
       modules:
       - "module_name1:stream1"
       - "module_name2:stream1"
 
+**Exactly one non-empty list of "modules" or "packages" must be provided. If
+both or none are defined build will fail.**
+
 Behavior Changes
 ----------------
 
 During an OSBS build, if ``container.yaml`` exists **and** ``compose`` key is defined:
 
 - OSBS orchestrator build will request a compose from ODCS based on specified
   parameters in ``compose`` section. Once compose completes, build resumes and
-  the URL for the generated compose is passed to worker builds via
-  ``yum_repourls`` build parameter. *NOTE: If odcs-signed parameter was set to
-  false, the compose request will ignore the sigkeys provided in
-  container.yaml.*
+  the compose ID is passed to the worker builds via the new ``compose_ids``
+  build parameter. *NOTE: If odcs-signed parameter was set to false, the compose
```

Comment:
hm... did I get lost in the diffs? Where should this parameter be set? Will this be a value in container.yaml?

Or this is meant as `intent` not defined in container.yaml?

Reply:
`odcs-signed` is ultimately a CLI parameter, that gets propagated to koji task, and eventually osbs create build api.

Reply:
Gotcha, no questions then.